posts/bayesian-model-comparison/index.qmd

```diff
@@ -291,7 +299,7 @@ Here we will say that a difference of 1% in performance between the two models m
 #| code-fold: true
 #| code-summary: code for `get_t_dist_posterior_probs`
```

Comment:
Do we like "code for ...", is there an alternative that could be better? In any case maybe these should be capitalised, i.e. "Code for ..."